apps/sv/frontend/src/utils/governance.ts

```diff
@@ -34,7 +35,7 @@ export const actionTagToTitle = (amuletName: string): Record<SupportedActionTag,
   SRARC_UpdateSvRewardWeight: 'Update SV Reward Weight',
 });
 
-export const createProposalActions = [
+export const createProposalActions: { name: string; value: SupportedActionTag }[] = [
```

Comment:
@ray-roestenburg-da notice the `SupportedActionTag` type here which forces you to use the right values at compile time.

Reply:
nice!